progress_observer/progress_observer.go

```diff
@@ -66,11 +72,15 @@ func ObserveIndexers(sourceUrl, targetUrl string, m *metrics.Metrics) error {
 		log.Fatal(err)
 	}
 
+	m.RecordCount(len(sources), sourceName, "", metrics.TotalCount)
```

Comment:
Empty target is going to create the tag still, with empty value. Should there be an `if` statement to avoid this [here](https://github.com/ipni/index-observer/blob/63220fa60b5fac1d68f6e696c1b29e236621c353/progress_observer/metrics/metrics.go#L100)? or even better refactor metrics and expose a new function for collencting what this line collects that does not take target.

Reply:
Good point - changed to not add target tag if it's empty